tests/countries/test_turks_and_caicos_islands.py

```
@@ -0,0 +1,75 @@
+            ("2025-03-10", "Commonwealth Day"),
+            ("2025-04-18", "Good Friday"),
+            ("2025-04-21", "Easter Monday"),
+            ("2025-05-26", "JAGS McCartney Day"),
+            ("2025-06-09", "King's Birthday"),
+            ("2025-08-01", "Emancipation Day"),
+            ("2025-09-26", "National Youth Day"),
+            ("2025-10-13", "National Heritage Day"),
+            ("2025-11-28", "National Day of Thanksgiving"),
+            ("2025-12-25", "Christmas Day"),
+            ("2025-12-26", "Boxing Day"),
+        )
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)